File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {

    int prev=0,crr=1;
    while(crr<numsSize){
        if(nums[crr]==nums[prev]){
            crr++;
        }
        else{
            nums[prev+1]=nums[crr];
            prev++;
            crr++;
        }
    }
    return prev+1;

}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int prev=0;
        int crr=1;
        while(crr<nums.length){
            if(nums[prev]==nums[crr]){
                crr++;
            }
            else{
                nums[prev+1]=nums[crr];
                prev++;
                crr++;
            }
        }

        return prev+1;
    }
}